config/billing_rates.yml

```diff
@@ -30,6 +30,7 @@
 - { id: 1172b71c-4c1f-8978-aa10-b9e73822a8e4, resource_type: PostgresStandbyStorage,  resource_family: standard,        location: leaseweb-dal13, unit_price: 0.000002810 }
 - { id: d772d0aa-0b40-4b7a-aceb-72f6211f7cad, resource_type: GitHubRunnerMinutes,     resource_family: standard-2,      location: global,         unit_price: 0.0008 }
 - { id: d885c82d-b139-40d5-890f-de3720896b3b, resource_type: GitHubRunnerMinutes,     resource_family: standard-4,      location: global,         unit_price: 0.0016 }
+- { id: 27d84309-e39d-8d78-b603-7632dafe374e, resource_type: GitHubRunnerMinutes,     resource_family: standard-6,      location: global,         unit_price: 0.0320 }
```

Comment:
Without knowing the context, it is quite weird that the standard-6 costs more than standard-8 for the same resource_type. I think we should use another resource_type; GithubGpuRunnerMinutes for example.